completion/_autocomplete.history_lines

```diff
@@ -1,17 +1,26 @@
 #autoload
 
 _autocomplete.history_lines() {
+  local -a just_values
+  local -A values_sorted
+  local -A keys
   local tag=history-lines
   _tags $tag
   _tags && _requested $tag ||
     return 1
-
+  for key val in "${(@kv)history}"; do
```

Comment:
I wouldn't do that. `$history` can be _HUGE._ You shouldn't sort the whole history, just the matching values.

Also, wasn't the point of #129 to eliminate duplicates? Try to focus on that.

And you don't need to eliminate duplicates from the entire history. Just from the search results.